accepted/build/dotnet-patch-check.md

```diff
@@ -0,0 +1,27 @@
+# Getting latest .NET Core Runtime patch information
+
+## Goal
+
+When patches are released for the .NET Core runtime, developers should not need to install an entirely new .NET Core SDK.  They should be able to install only the updated runtime as needed, and the existing SDK should have a way of discovering that new patches should be used for self-contained applications.
+
+Eventually we hope to be able to stop producing new releases of the .NET Core SDK whenever there is a patch to the runtime.  This proposal is a step in that direction, however a lot more will be needed to enable that.
```

Comment:
> Eventually we hope to be able to stop producing new releases of the .NET Core SDK whenever there is a patch to the runtime. 

This is the wrong framing and sends the wrong message, IMO. In fact, I think that this statement is barely relevant to the overall topic. Instead, this scenario should be about a single topic:

- enabling you to publish a self-contained app with the latest runtime, w/o updating your SDK.

In addition (but not part of this document), we should have a promise that we always give users the latest runtime whenever they use our SDK distributions.